src/Evolution/VariableFixing/FixToAtmosphere.cpp

```diff
@@ -19,23 +19,42 @@ namespace VariableFixing {
 template <size_t Dim>
 FixToAtmosphere<Dim>::FixToAtmosphere(const double density_of_atmosphere,
                                       const double density_cutoff,
+                                      const double transition_density_cutoff,
+                                      const double max_velocity_magnitude,
                                       const Options::Context& context)
     : density_of_atmosphere_(density_of_atmosphere),
-      density_cutoff_(density_cutoff) {
+      density_cutoff_(density_cutoff),
+      transition_density_cutoff_(transition_density_cutoff),
+      max_velocity_magnitude_(max_velocity_magnitude) {
   if (density_of_atmosphere_ > density_cutoff_) {
     PARSE_ERROR(context, "The cutoff density ("
                              << density_cutoff_
                              << ") must be greater than or equal to the "
                                 "density value in the atmosphere ("
                              << density_of_atmosphere_ << ')');
   }
+  if (transition_density_cutoff_ < density_of_atmosphere_ or
+      transition_density_cutoff_ > 10.0 * density_of_atmosphere_) {
```

Comment:
This upper-bound constraint of 10*rho_atmo should be documented in the option struct at least but probably also in the main doxygen for the class